events.json

```diff
@@ -1,183 +0,0 @@
-[{
```

Comment:
I'm starting to think the json of the site should probably be in a separate repo since it adds a decent bloat to the site.